examples/text_classification/vocab.py

```diff
@@ -0,0 +1,39 @@
+import logging
```

Comment:
same as above. this example is to show how to build vocab so it should be under examples directory.